keyboards/handwired/mingwahlu/mingnumpad/info.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
This file is not the right format. This is the VIA JSON which should be added to the VIA repo.

Reply:
fixed the format to the correct layout that includes X and Y.

eg.
```js
{
   "label": "k00",
   "x": 0,
   "y": 0
}
```

Reply:
have fixed all the json files and the pr lint now works.